plaso/lib/lexer.py

```diff
@@ -12,26 +12,22 @@
 
 from plaso.lib import errors
 
-# TODO: Update docstrings in this file.
-# pylint: disable=missing-type-doc,missing-return-type-doc,missing-return-doc
-# pylint: disable=missing-param-doc
-
 
 class Token(object):
-  """A token action."""
+  """An event filter parser token."""
 
   def __init__(self, state_regex, regex, actions, next_state, flags=re.I):
-    """Initializes the token object.
+    """Initializes an event filter expressions parser token.
 
     Args:
-
-      state_regex: If this regular expression matches the current state this
-                   rule is considered.
-      regex: A regular expression to try and match from the current point.
-      actions: A command separated list of method names in the Lexer to call.
-      next_state: The next state we transition to if this Token matches.
-      flags: re flags.
+      state_regex (str): regular expression that is considered when the current
+          state matches this rule.
+      regex (str): regular expression to try and match from the current point.
```

Comment:
This is an re too, not a string.

Reply:
this will be made protected in a later change